showpass/Showpass-E2E-tests
# Fixes and improvements 04Dec.2023

**Description:**
Tests related to the Membership groups and levels began failing yesterday and continued failing today all of a sudden.
Applied a couple code fixes to bring it back to normal.

**How to test:**
Run all specs and ensure the tests pass.

**Test execution results:**[
](https://cloud.cypress.io/projects/e9axi7/runs/267/overview?roarHideRunsWithDiffGroupsAndTags=1)
